### PR TITLE
Adding a task to clean files and/or directories

### PR DESCRIPTION
**Pull request by oanhnn**
| Q                    | A
| -------------------| ---
| Bug fix?          | no
| New feature?  | yes
| BC breaks?     | no
| Deprecations? | no
| Fixed tickets    | -

As comment in PR #453, in this PR:
* Added task to clean up files and/or directories

### CODE REVIEW

File: recipe/common.php [hunk 1]
```diff
@@ -15,6 +15,8 @@
 set('writable_dirs', []);
 set('writable_use_sudo', true); // Using sudo in writable commands?
 set('http_user', null);
+set('clear_paths', []);
+set('clear_use_sudo', true); // Using sudo in writable commands?
```
Comment (vanquang9387) [inline recipe/common.php]: @oanhnn should edit the comment

**Comment by jverdeyen** [comment]
Can I change this setting for each server/stage indivually?

**Comment by oanhnn** [comment]
@jverdeyen I can use environment variables instead of parameters, it is maybe change setting for each server/stage individually. But, i want create common task same `deploy:writable` task :smile:

**Comment by jverdeyen** [comment]
Yes, writeable dirs are bounded to the application, but cleaning controllers (or files) can also change per stage/server.

@elfet what's your idea about this feature?

**Comment by vanquang9387** [comment]
I think we may set both common var & env var for flexibility.
For example:

In common recipe:
```
set('clear_paths', [...]);
```

In server/stage config:
```
server(...)->env('clear_paths', [...]);
```

Then in deploy:clean:
```
    $paths = get('clear_paths');
    // Use env var if exists
    if (env()->has('clear_paths')) {
        $paths = env('clear_paths');
    }
...
```

**Comment by elfet** [comment]
@oanhnn rebase please.

**Comment by oanhnn** [comment]
@elfet Please merge :smile:

**Pull request merged**